sonar/theme/templates/sonar/footer.html

```diff
@@ -54,8 +50,9 @@
       <div class="col text-center p-0">
         <div class="d-lg-flex flex-row">
           <div class="col-lg text-center text-lg-right">
-            {% set link = 'https://www.rero.ch/de/products/sonar' if current_i18n.locale.language == 'de' else 'https://www.rero.ch/produits/sonar' %}
-            {% trans %}Powered by <a href="{{ link }}" target="_blank" rel="noopener">SONAR</a>{% endtrans %}
+            {% trans %}Powered by <a href="https://www.rero.ch/produits/sonar" target="_blank" rel="noopener">SONAR</a>{% endtrans %} |
+            {{ _('Version') }} <a href="https://github.com/rero/sonar">{{ version }}</a> |
```

Comment:
You can perhaps include the right tag to the github link.

Reply:
Done